### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by BigDaddyonTech** [quodlibet/quodlibet#2172]
I installed on Manjaro Gnome and opened it up. It crashed just after adding my music directory and now will not open without crashing

[Dump_20161224_181714.txt](https://github.com/quodlibet/quodlibet/files/671993/Dump_20161224_181714.txt)

**Comment by lazka** [comment]
Thanks. You can disable the library scan on start in the preferences to prevent it for now.

**Comment by lazka** [comment]
Can you somehow figure out which file is causing this and send it to me? (reiter.christoph@gmail.com)
You can use Ex Falso to look through your music until you hit the error for example.

**Comment by BigDaddyonTech** [comment]
I have gone through my music and removed some lyric files and images as well that I no longer needed. There is now only audio files in there and it actually scans for a longer period of time and almost completely finishes before it crashes. I still can't find the file or files it is hanging up on. I will try to keep looking.

**Comment by lazka** [comment]
Thanks. Using exfalso and view folders one by one might work. Sadly 3.7 does not print the path to the problematic file, will be fixed in 3.8.

**Comment by BigDaddyonTech** [comment]
I might try to use exfalso but it will have to wait until my vacation starting Jan 1st because my music library has over 4500 songs so it might take some time to find the problem child.

**Comment by lazka** [comment]
3.8.0 is now in Arch which should print out the problematic file.

**Comment by lazka** [comment]
Any news on this?

**Comment by BigDaddyonTech** [comment]
I apologize for not trying to dig into it any further. I have been having a rough couple of weeks with switching distributions. In between, I did try to dig in for the offending files but I couldn't seem to pinpoint it. Maybe it's my lack of knowing how to pinpoint the file but either way, I could not find the file that was causing the issue. 
I will say that it was still crashing when I open it up. I just reinstalled my system a day ago, so I will install the program again and see if it happens.
